test/basic.html

```diff
@@ -206,6 +206,77 @@
         });
       });
 
+      suite('accessibility', function() {
+        var LEFT = 37;
+        var RIGHT = 39;
+        var tabs;
+
+        setup(function () {
+          tabs = fixture('basic');
+          ensureDocumentHasFocus();
```

Comment:
Actually, it seems like this won't completely get rid of the need for `ensureDocumentHasFocus()`. The tests are actually about what happens in response to the item getting focus, so it won't be enough to just check that `focus` was called on the element.